admin/product_types.php

```diff
@@ -208,7 +211,7 @@
 
             switch ($action) {
               case 'layout_edit':
-                $heading[] = ['text' => '<h4>' . $cInfo->configuration_title . '</h4>'];
+                $heading[] = ['text' => '<h4>' . $cInfo->configuration_title . '</h4>', ];
```

Comment:
Um ... ?

Reply:
Yeah, a little on the extreme end of refactoring. Was aiming for arrays to end with a comma. Realizing though that lose a little of the desired benefit if everything is on one line rather than supporting the addition of a new line with little edit...

Reply:
In PHP the reason for adopting the PSR standard of ending multiline arrays with a comma was so that `git diff` operations would be cleaner whenever changes were made to multiline definitions.
For single-line array declarations/insertions, no trailing comma is needed, and is not in the spec. (And if another line were to be added to the single-line array statement, it would become a multi-line statement, which would affect the line in multiple ways, which would then be relevant. Until then, terse is desirable.)